pkg/services/config.go

```diff
@@ -356,7 +357,7 @@ func New(ctx context.Context, config Config) (*Services, error) {
 		authenticators = union.New(authenticators, authn.NewNoAuth(gatewayClient))
 	}
 
-	if config.EmailServerName != "" {
+	if config.EmailServerName != "" && config.EnableSMTPServer {
```

Comment:
I don't think this works because of this line: https://github.com/obot-platform/obot/blob/6d035afc3b0b112c2aa8b58b2209f25224876eaa/pkg/emailtrigger/emailtrigger.go#L46

Essentially, the email will never be sent if `config.EnableSMTPServer` is set, but `config.EmailServerName` is not.